src/mSupplyMobileApp.js

```diff
@@ -21,11 +20,15 @@ import {
 } from 'react-native';
 
 import { Scheduler } from 'sussol-utilities';
+import { NavigationActions } from 'react-navigation';
+import globalStyles, {
```

Comment:
nit: I preferred the other format with styles being the last import

Reply:
I also agree, though I think prettier/eslint setup is currently going to force an order of:
1. Node module imports alphabetically
2. Relative imports alphabetically

Reply:
Yes this order is forced by eslint